recipes/b4/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "b4" %}
+{% set version = "0.14.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://mirrors.edge.kernel.org/pub/software/devel/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 62e739777d7963c72acf65b06a3f80762162e62eaabb53b5129a73291f2d4fa1
+
+build:
+  skip: True  # [win or py<38]
```

Comment:
Looks like this one can be OS-noarch. See https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies for more info.

Reply:
The build steps runs some Bash scripts, how do I restrict it to only building on Unix platforms?

conda-forge-linting says I can't use `skip: True # [win]` with `noarch: python`
The virtual packages you linked seem to be for run-time dependencies. I tried setting `__unix` as a host requirement, but the pipeline still tries to build for Windows.

Reply:
> The build steps runs some Bash scripts, how do I restrict it to only building on Unix platforms?

You don't. If you follow the documentation above and add the __unix dependency on run this package may fail to build on Windows b/c it won't be installable, but that is OK.

Reply:
I added `__unix` to the run dependeny, thank you